### PR TITLE
AO3-6418 Add Wattpad to import warning

### PR DESCRIPTION
**Pull request by albinantti**
* [✅] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [✅] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [❌] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
     * No functionality changes
* [✅] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [✅] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6418

## Purpose

Adds Wattpad.com to the banner at the top of https://archiveofourown.org/works/new?import=true

## Testing Instructions

Attempt to import a new work.

## References

None

## Credit

albinantti (He/him)

### CODE REVIEW

**Review by sarken — CHANGES_REQUESTED** [review]
Hi, albinantti!

Thank you so much for this pull request. It looks like it just needs a tiny punctuation fix.

I've updated the Jira issue status to In Review, so no one will mistakenly create a duplicate pull request. If you'd like the ability to comment on, assign, and transition issues in the future, you're welcome to create a Jira account! You can just reply here with the account name and we'll set up the permissions for you. (It makes things a bit easier for us on the organizational side if the Full Name on your Jira account either closely matches the name you'd like us to credit in the release notes or includes it in parentheses, e.g. "Nickname (PREFERRED NAME).")

Thanks again for contributing! If you have any questions, you can contact us at otw-coders@transformativeworks.org.

File: app/views/works/new_import.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <%= error_messages_for :work %>
 <div class="notice">
   <p class="important">
-    <%= ts("Please note! Fanfiction.net and Quotev.com do not allow imports from their sites.") %>
+    <%= ts("Please note! Fanfiction.net, Wattpad.com and Quotev.com do not allow imports from their sites.") %>
```
[the remark this answers is not in the document]
Comment (albinantti) [reply app/views/works/new_import.html.erb]: Didn't realise there was an oxford comma, it's been added :)

**Review by sarken — APPROVED** [review]
Thank you! I've updated your Jira so you can claim, transition, and comment on issues in the future.

**Comment by sarken** [comment]
Hi again! I just wanted to double check: do you prefer to be credited in our release notes as Albin Antti or albinantti? Either is fine, but I want to make sure I get it right.

**Comment by albinantti** [comment]
albinantti is fine 🥳

**Pull request merged**